Учебный проект: Пошаговый слайдер, часть 2 - не смог пройти все проверки

ArtemMiroshnichenkoI opened the pull request.

ArtemMiroshnichenkoI committed:
Учебный проект: Пошаговый слайдер, часть 2
 - не проходит три теста

ArtemMiroshnichenkoI committed:
Учебный проект: Пошаговый слайдер, часть 2
 - не проходит три теста

ArtemMiroshnichenkoI committed:
Вот конечная версия в которой я не понимаю, что не так

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

ArtemMiroshnichenkoI committed:
Учебный проект: Пошаговый слайдер, часть 2
- решено

jsru commented:
Добавляю преподавателя (@stas-dolgachov) для код-ревью.

ArtemMiroshnichenkoI committed:
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20210722_l1nq

ArtemMiroshnichenkoI committed:
Учебный проект: Иконка корзины
Учебный проект: Иконка корзины
ошибка: иконка должна смещаться на 10px левее главного контейнера
но мне кажется все и так работает

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

ArtemMiroshnichenkoI committed:
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20210722_l1nq

dolgachio commented on 7-module/4-task/index.js:
```diff
@@ -1,4 +1,135 @@
+    this.elem.querySelector(".slider__value").innerHTML = this.findSpan;
+  }
+
+  onClick = (event) => {
```
👍

dolgachio commented on 7-module/4-task/index.js:
```diff
@@ -1,4 +1,135 @@
+
+        let percentForMove = mouseX / this.elem.getBoundingClientRect().width * 100;
+
+        this.elem.querySelector('.slider__thumb').style.left = `${mouseX / this.elem.getBoundingClientRect().width * 100}%`;
```
Это выражение можно только раз рассчитать, а потом присваивать из переменной

dolgachio commented on 7-module/4-task/index.js:
```diff
@@ -1,4 +1,135 @@
+    document.documentElement.addEventListener("pointermove", forMove);
+
+
+    let forPointerUp = (event) => {
```
👍

dolgachio commented on 8-module/1-task/index.js:
```diff
@@ -39,6 +40,33 @@ export default class CartIcon {
+      this.elem.style.marginRight = "";
+    }
+
+    // console.log(document.querySelector(".container").children[0].getBoundingClientRect().right + this.elem.getBoundingClientRect.width + 10);
```
Закомментированый код обычно лучше удалять

dolgachio commented on 7-module/4-task/index.js:
```diff
@@ -1,4 +1,135 @@
+    this.elem.querySelector('.slider__thumb').ondragstart = () => false;
+    this.elem.classList.add("slider_dragging");
+
+    let forMove = (event) => {
```
👍

ArtemMiroshnichenkoI committed:
8.1 и 8.2 решены

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

ArtemMiroshnichenkoI committed:
8.1 и 8.2 решены(без закоммментированного кода

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

ArtemMiroshnichenkoI committed:
8.1 и 8.2 (окончательное решение)

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

jsru commented:
Добавляю преподавателя (@stas-dolgachov) для код-ревью.
